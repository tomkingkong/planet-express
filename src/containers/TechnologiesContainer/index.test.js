import React from 'react';
import { shallow } from 'enzyme';
import { 
  TechnologiesContainer, 
  mapStateToProps } from '.';

describe('TechnologiesContainer Container', () => {
  it('should match snapshot', () => {
    const mockProps = {
      technologies: [{}]
    };
    const wrapper = shallow(<TechnologiesContainer {...mockProps} />);
    expect(wrapper).toMatchSnapshot();
  });

  describe('mapStateToProps', () => {
    it('should be provided with a technologies array from props', () => {
      const mockState = {
        spaceEvents: [{}, {}],
        technologies: [],
        exoplanets: [],
        journey: {},
        news: []
      }
      const expected = { technologies: [] };
      const results = mapStateToProps(mockState);
      expect(results).toEqual(expected);
    });
  });
});