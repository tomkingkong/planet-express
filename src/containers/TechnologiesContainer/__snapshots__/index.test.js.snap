// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TechnologiesContainer Container should match snapshot 1`] = `
<div
  className="Technologies"
>
  <header
    className="technologies__title"
  >
    <h4>
      SpaceLab
    </h4>
  </header>
  <Component
    key="0"
  />
</div>
`;
